json_journal/filters.py

```diff
@@ -0,0 +1,4 @@
+def filter_balances_to_expenses(all_balances):
+    expenses = [[account, all_balances[account]] for account in all_balances if account.startswith("expense:")]
+    expenses_dict = dict(expenses)
```

Comment:
Would [dictionary comprehension](https://stackoverflow.com/questions/3420122/filter-dict-to-contain-only-certain-keys) be nice here?

Reply:
done